soccer/gameplay/plays/offense/cross_kick.py

```diff
@@ -0,0 +1,272 @@
+import robocup
+import constants
+import play
+import skills
+import tactics
+import evaluation
+import main
+import enum
+import behavior
+
+class CrossKick(play.Play):
+
+    class State(enum.Enum):
+    	collecting = 0;
```

Comment:
Can you add a description for all the states. With all the different transitions and states the readability is a little tough.